src/internal/sdata/sdata.go

```diff
@@ -74,8 +76,8 @@ func MaterializeSQL(tw TupleWriter, rows *sql.Rows) (*MaterializationResult, err
 	}, nil
 }
 
-// NewTupleFromSQL returns a new Tuple based on column types from the sql package.
-func NewTupleFromSQL(colTypes []*sql.ColumnType) Tuple {
+// TODO: replace this with makeTupleElement
+func newTupleFromSQL(colTypes []*sql.ColumnType) Tuple {
```

Comment:
maybe rename this as it's ambiguous now, `newTupleFromColumns` or something
What does "replace" mean here, like we'd go through the `GetTableInfoTx` in all cases?